### ISSUE TITLE
Enable stacked cards on Kusama

### ISSUE THREAD

**Issue by daiagi** [kodadot/nft-gallery#7296]
Query for token entities instead of nft entities
similar to how it's done on AssetHub

goes after
- #7269 
- https://github.com/kodadot/rubick/pull/274

**Comment by daiagi** [comment]
I'll do this once token entity is enabled on rubick

**Comment by daiagi** [comment]
@vikiival newest change on rubick (TokenEntity) isn't deployed. is it?

**Comment by vikiival** [comment]
> @vikiival newest change on rubick ([TokenEntity](https://github.com/kodadot/rubick/pull/274)) isn't deployed. is it?

Blocked by #7373

**Comment by daiagi** [comment]
@vikiival still blocked?

**Comment by vikiival** [comment]
> @vikiival still blocked?

Yes but first I would resolve the issues with tokenentitty query on frontend

https://pastebin.com/LsFMXEMb

It was one of the reasons why squid failed
